src/components/PopOverCanvas.tsx

```diff
@@ -0,0 +1,122 @@
+import React, { useRef, useState } from 'react';
+
+const PopOverCanvas = () => {
+  const referenceElRef = useRef(null);
+  const [showPopover, setShowPopover] = useState(false);
+
+  const handlePopoverToggle = () => {
+    setShowPopover(!showPopover);
+  };
+
+  return (
+    <div>
+      <div className="tds-headline-02 tds-u-pb1">Popover</div>
+      <div className="tds-headline-03 tds-u-pb1">Canvas</div>
+
+      <div className="popover-container">
+        <tds-button
+          aria-label="menu"
+          only-icon
+          id="trigger"
+          type="ghost"
+          size="sm"
+          ref={referenceElRef}
+          onClick={handlePopoverToggle}
+        >
+          <tds-icon
+            slot="icon"
+            className="tds-btn-icon"
+            size="16px"
+            name="kebab"
+          ></tds-icon>
+        </tds-button>
+      </div>
+
+      <tds-popover-canvas
+        placement="auto"
+        selector="#trigger"
+        className={`tds-u-p2 ${showPopover ? 'tds-popover-canvas-show' : ''}`}
```

Comment:
I don't think this class is meant to be tampered with by the users, they should use the show:boolean prop to toggle show/hide state of the popover.